torch/csrc/distributed/autograd/context/dist_autograd_container.h

```diff
@@ -60,6 +60,9 @@ class TORCH_API DistAutogradContainer {
   // can be generated by this worker.
   int64_t getMaxId();
 
+  // retrieves the worker ID for this autograd context
```

Comment:
There is one single worker id for each node, we don't have a worker id for each context. Lets just say we retrieve the worker id for this node.